### PR TITLE
Bo 2273/volume prices for product offers

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

https://spryker.atlassian.net/browse/BO-3356

- Added a feature overview and a feature walkthrough
- Updated the BO UG for product offers

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/features/202108.0/marketplace-merchant-portal-product-offer-management-feature-overview.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Merchants create product offers based on the products that exist in the marketplace. When creating an offer, they can see full information about products, including the number of existing offers for each of them.
+
+One merchant can create multiple offers for the same product.
+
+When creating an offer, they can define the following settings:
+
+|SETTING|DESCRIPTION|
+|---|---|
+| Status: Active or Inactive |
+| Merchnat SKU | Unique identifier of the merchants' offer. |
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202108.0/marketplace-merchant-portal-product-offer-management-feature-overview.md:21]: Unique identifier of the merchants' offer is Offer Reference, the Merchant SKU is when the Merchant is using a different SKU in his/her system than the one used in the marketplace for this product

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md [hunk 2]
```diff
@@ -17,51 +17,37 @@ Review the reference information in this article before you start, or just look
 To create a product offer:
 
 1. Click **Add Offer**.
-2. From the list of concrete products, select the necessary product you want to create an offer for, hover over the three dots, or just click the line and then click **Create Offer**.
+2. From the list of concrete products, select the  product you want to create an offer for.
 3. In the drawer, fill the necessary fields and click **Create**.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md:21]: Consider adding information from this PR: https://github.com/spryker/spryker.github.io/pull/211/files Specifically, the info from lines 20-30 (no need to include the detailed info about the fields that you have in the reference information section of course). "Fill the necessary fields" is not a very helpful step, and there is no information about such fields as for example "Always in stock" or "Offer status". Consider adding them in the reference information or here

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md [hunk 3]
```diff
@@ -83,41 +69,59 @@ You can sort the existing product offers list by:
...
+Using the search field in the top right corner, you can search product offers by:
 
 - Offer reference
 - merchant reference
 - product SKU
 
-### Create offer page
+### Create Offer page
+
+On the *Create Offer* page, you see the list of product you can create an offer for. You can filter, sort and search the products.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md:82]:
```suggestion
On the *Create Offer* page, you see the list of products you can create an offer for. You can filter, sort and search the products.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md [hunk 4]
```diff
@@ -83,41 +69,59 @@ You can sort the existing product offers list by:
...
+
+#### Filtering products
+
+Using the filtering menu at the top of the page, you can filter products by:
+
+- Existing offers
+- Visibility
+
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md:90]: Please specify the field names where you can filter offers by existing and visibility. It's not obvious, as the fields for filtering are called just "offers" and "status"

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Review by lenadoc — APPROVED** [review]

**Review by AdvaBD — APPROVED** [review]

**Pull request merged**